<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenie</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="sidePanel" native="true">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rotation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Distance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="distanceSlider">
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="rotationSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="reflectionGroupBox">
         <property name="title">
          <string>Reflection</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="reflectionOffsetSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Opacity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="reflectionOpacitySlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="backgroundColorGroupBox">
         <property name="title">
          <string>Background</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Red</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="redSlider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSpinBox" name="redSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="greenSlider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="greenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Blue</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="blueSlider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="blueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>HTML</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="htmlBGColorLineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="maxLength">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>453</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="recentFilesMenu">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
    </widget>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="saveAsTemplateAction"/>
    <addaction name="separator"/>
    <addaction name="exportAction"/>
    <addaction name="separator"/>
    <addaction name="recentFilesMenu"/>
    <addaction name="separator"/>
    <addaction name="closeAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="menu_Insert">
    <property name="title">
     <string>&amp;Insert</string>
    </property>
    <addaction name="addImageAction"/>
    <addaction name="addTemplateAction"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="toggleFullScreenAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="deleteAction"/>
    <addaction name="separator"/>
    <addaction name="selectAllAction"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Insert"/>
   <addaction name="menu_View"/>
   <addaction name="windowMenu"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addImageAction"/>
   <addaction name="addTemplateAction"/>
   <addaction name="separator"/>
   <addaction name="toggleFullScreenAction"/>
  </widget>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/application-exit.png</normaloff>:/img/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="addImageAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/insert-image.png</normaloff>:/img/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Image ...</string>
   </property>
   <property name="toolTip">
    <string>Add Image</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/document-open.png</normaloff>:/img/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/document-save.png</normaloff>:/img/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/document-save-as.png</normaloff>:/img/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As ...</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>&amp;Export Image</string>
   </property>
   <property name="toolTip">
    <string>Export Image ...</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="addTemplateAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/insert-template.png</normaloff>:/img/insert-template.png</iconset>
   </property>
   <property name="text">
    <string>Template</string>
   </property>
   <property name="toolTip">
    <string>Add Template</string>
   </property>
  </action>
  <action name="toggleFullScreenAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/view-fullscreen.png</normaloff>:/img/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
   <property name="toolTip">
    <string>View Full Screen</string>
   </property>
  </action>
  <action name="selectAllAction">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/edit-cut.png</normaloff>:/img/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/edit-copy.png</normaloff>:/img/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/edit-paste.png</normaloff>:/img/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/edit-delete.png</normaloff>:/img/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="newAction">
   <property name="icon">
    <iconset resource="../../Resources/Resources.qrc">
     <normaloff>:/img/document-new.png</normaloff>:/img/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="saveAsTemplateAction">
   <property name="text">
    <string>Save As &amp;Template ...</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
